test/boost_ext/CompressedStorageComparison.hpp

```diff
@@ -41,22 +41,19 @@ struct CompressedStorage
       ++*this;
       return old;
     };
-    ConstForwardIterator operator==(ConstForwardIterator const &o)
-    {
-      return i == o.i;
-    }
-    ConstForwardIterator operator!=(ConstForwardIterator const &o)
-    {
-      return !(*this == o);
-    }
+    bool operator==(ConstForwardIterator const &o) { return i == o.i; }
+    bool operator!=(ConstForwardIterator const &o) { return !(*this == o); }
     value_type operator*()
     {
       return {p->values.data() + p->offsets[i],
               p->values.data() + p->offsets[i + 1]};
     }
   };
   ConstForwardIterator cbegin() const { return {0, this}; }
-  ConstForwardIterator cend() const { return {offsets.size(), this}; }
+  ConstForwardIterator cend() const
+  {
+    return {static_cast<index_type>(offsets.size()), this};
```

Comment:
Was this causing a problem?

Reply:
Yes, I got an error for the narrowing implicit cast.

Reply:
Please cast in cbegin as well for consistency